src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -575,14 +576,32 @@ void reorder_inputs::run(program& p, layout_optimizer& lo, reorder_factory& rf)
         }
     };
 
+    const auto reorder_input_fully_connected = [&p, &lo, &rf](typed_program_node<fully_connected>& fc_node) {
+        auto& weights = fc_node.weights();
+        auto& input = fc_node.input();
+        auto weights_layout = weights.get_output_layout();
```

Comment:
this is not being used, is it?

Reply:
(saw above comment after I left this.. seems that the weights_layout is left after removal of the above check :)